phobos/core/robot.py

```diff
@@ -2530,12 +2532,15 @@ def derive_model_dictionary(root, name='', objectlist=[]):
             # joint may be None if link is a root
             # to prevent confusion links are always defining also joints
             jointdict = deriveJoint(link, logging=True, adjust=True)
-            # log("  Setting joint type '{}' for link.".format(jointdict['type']), 'DEBUG')
+            log.debug("  Setting joint type '{}' for link.".format(jointdict['type']))
             # first check if we have motor information in the joint properties
             # if so they can be extended/overwritten by motor objects later on
             if '$motor' in jointdict:
                 motordict = jointdict['$motor']
-                # at least we need a type property
+                if 'mimic_motor' in motordict:
+                    motordict['type'] = 'mimic'
+                    motordict['name'] = motordict['mimic_motor'] + '_mimic'
+                # at least we need a type property, TODO WIESO ? DESHALB FLIEGEN ALLE 'mimic_motor', HOTFIXED
```

Comment:
Das sieht etwas dubios aus... Let's talk about this.

Reply:
Das ist passiert um sherpat zum Laufen zu bringen. Da hatten die mimic_motoren keinen "type" Eintrag aber dafür "mimic_motor". Mit dem codepiece kann man so ein Format denke ich einfangen und alles andere stumpf weiterlaufen lassen wie vorher. Man kann das natürlich ein bisschen schönes machen aber die Idee ist denke ich richtig.

Reply:
geklärt